setup.py

```diff
@@ -105,7 +105,7 @@ def initialize_options(self) -> None:
     # 'googleapiclient.discovery.build':
     #
     # https://github.com/googleapis/google-auth-library-python/issues/190
-    'google-api-python-client>=1.8.0,<1.9.0',
+    'google-api-python-client>=1.8.0',
```

Comment:
maybe a `<2` is in order here for future proofing?

Reply:
There's a lot of debate about if upper version bounds cause more harm than good. My feeling is that if we don't know that versions >2 will cause problems, we shouldn't add a restriction. Without a comment to add context, we'd quickly lose institutional knowledge about why the bound was there in the first place and it'd become some inscrutable constraint for future us to puzzle over. :)